x-pack/plugins/fleet/common/openapi/paths/service_tokens.yaml

```diff
@@ -0,0 +1,24 @@
+post:
+  summary: Generate service tokens
+  tags: []
+  responses:
```

Comment:
could the response be more specific? It renders weirdly on swagger:
https://petstore.swagger.io/?url=https://raw.githubusercontent.com/elastic/kibana/330634f8f6be33bf3ad494b17167fbf107628bd1/x-pack/plugins/fleet/common/openapi/bundled.json#/default/post_service_tokens

Reply:
Yes, I'm trying to fix it. For some reason the schema is not getting read properly